docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -40,24 +40,45 @@ abstract class Client {
   // get all notifications for a subscription
   public abstract getNotificationHistory(params: {
     topic: string,
-  }): Promise<Record<number, NotifyNotificationRecord>>
+    // Default 10, max 50
+    limit?: number,
+    startingAfter?: string,
```

Comment:
`startingAfter` is a timestamp or number of messages?

Reply:
The ID (not message ID) of the last message on the previous page